frontend/src/components/workspaces/DashboardLogic/DashboardLogic.tsx

```diff
@@ -12,8 +12,8 @@ const DashboardLogic: FC<{}> = () => {
 
   const { loading, error, subscribeToMore } = useTenantQuery({
     variables: { tenantId: userId ?? '' },
-    notifyOnNetworkStatusChange: true,
     onCompleted: setData,
+    fetchPolicy: 'network-only',
```

Comment:
Nit. Could be probably defined as constant, if that makes sense to you.

Reply:
For typescript "network-only" is not a normal string but a special type that can have only a list of value, there is an implicit cast. Anyway is not a problem to declare it as a constant. i'm going to do it